addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -27,6 +31,86 @@ class PaymentAcquirer(models.Model):
              "authenticate the messages sent from Stripe to Odoo.",
         groups='base.group_system')
 
+    # === ACTION METHODS === #
+
+    def action_stripe_connect_account(self, menu_id=False):
+        """Create a Stripe Connect account and redirect the user to the next onboarding step.
+
+        If the acquirer is already enabled, close the current window. Otherwise, generate a Stripe
+        Connect onboarding link and redirect the user to it. If the link generation failed, redirect
+        the user to the acquirer form.
+
+        Note: self.ensure_one()
+
+        :param str onboarding: The module from which the onboarding was initiated
+        :return: The next step action
+        :rtype: dict
+        """
+        self.ensure_one()
+
+        if self.state == 'enabled':
+            action = {'type': 'ir.actions.act_window_close'}
+
+        # Account creation
+        connected_account = self._stripe_fetch_or_create_connected_account()
+
+        # Link generation
+        if menu_id is False:
+            menu_id = self.env.ref('payment.payment_acquirer_menu').id
+        account_link_url = self._stripe_account_link_url(connected_account['id'], menu_id)
+        if account_link_url:
+            action = {
+                'type': 'ir.actions.act_url',
+                'url': account_link_url,
+                'target': 'self',
+            }
+        else:
+            action = self.env['ir.actions.actions']._for_xml_id('payment.action_payment_acquirer')
+            action.update(views=[[False, 'form']], res_id=self.id)
+
+        return action
+
+    def action_stripe_create_webhook(self):
+        """Create a webhook using the Stripe API
+
+        Note: This action does only work for instance using a public URL and doesn't handle local
+        endpoint.
+        """
+        self.ensure_one()
+
+        if self.stripe_webhook_secret or \
+           not self.stripe_secret_key:
+            if self.stripe_webhook_secret:
+                message = _('Your Stripe Webhook is already set up.')
+            elif not self.stripe_secret_key:
+                message = _('You can''t create a Stripe Webhook without a Stripe Secret Key set.')
+            action = {
+                'type': 'ir.actions.client',
+                'tag': 'display_notification',
+                'params': {
+                    'message': message,
+                    'sticky': False,
+                    'type': 'danger',
+                }
+            }
+        else:
+            webhook = self._stripe_make_request(
+                'webhook_endpoints',
+                payload={
+                    'url': self._get_stripe_webhook_url(),
+                    'enabled_events[]': WEBHOOK_HANDLED_EVENTS,
+                    'api_version': API_VERSION,
+                }
+            )
+            self.stripe_webhook_secret = webhook.get('secret')
+            action = True
+        return action
```

Comment:
There are some issues with this method:
- It's usually not recommended to do:
  ```python
  if A or B:
      if A:
      else B
   ```
   But here your `else` if in fact an `elif`, so you could just remove the parent `if`.
- Pro tip (not mandatory): Use single quotes (`'`) for technical strings and double quotes (`"`) for displayable strings. It helps with the readability and also doesn't require you to escape quotes in most cases (in technical strings, you rarely use `'` but you do in displayable strings).
- You return `True` instead of an action. It's valid but incoherent.
```suggestion
    def action_stripe_create_webhook(self):
        """ Create a webhook and return a feedback notification.

        Note: This action only works for instances using a public URL
        
        :return: The feedback notification
        :rtype: dict
        """
        self.ensure_one()

        if self.stripe_webhook_secret:
            message = _("Your Stripe Webhook is already set up.")
            notification_type = 'warning'
        elif not self.stripe_secret_key:
            message = _("You cannot create a Stripe Webhook if your Stripe Secret Key is not set.")
            notification_type = 'danger'
        else:
            webhook = self._stripe_make_request(
                'webhook_endpoints', payload={
                    'url': self._get_stripe_webhook_url(),
                    'enabled_events[]': WEBHOOK_HANDLED_EVENTS,
                    'api_version': API_VERSION,
                }
            )
            self.stripe_webhook_secret = webhook.get('secret')
            message = _("You Stripe Webhook was successfully set up!")
            notification_type = 'info'
        
        action = {
            'type': 'ir.actions.client',
            'tag': 'display_notification',
            'params': {
                'message': message,
                'sticky': False,
                'type': notification_type,
            }
        }
        return action
```